SNL_Instructions/module-1/instructions.md

```diff
@@ -2,6 +2,8 @@
 
 These instructions contain the hands-on lab modules for the Open Liberty Masterclass.  It is intended to be used in conjunction with taught materials, however, feel free to give it a try, even if you're not in a Masterclass.
 
+We strongly recommend using Google Chrome for the purpose of these labs.
```

Comment:
Can we do this before Module 1?

This may be too late and extra work for users to switch if we have this at this spot?

Reply:
I think it would be better to notify the users from the landing pages but we can also have this message in module 1 if people miss the first message.